com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -71,7 +71,37 @@ static CoreEditorUtils()
             return sb.ToString();
         }
 
+
         // UI Helpers
+
+        /// <summary>Creates a 1x1 <see cref="Texture2D"/> with a plain <see cref="Color"/></summary>
+        /// <param name="color">The color to fill the texture</param> 
+        public static Texture2D CreateColoredTexture2D(Color color)
+        {
+            Texture2D tex2 = new Texture2D(1, 1);
+            tex2.SetPixels32(new Color32[]{color});
+            tex2.Apply();
+            return tex2;
+        }
+
+        /// <summary>Changes the icon size of the <see cref="EditorGUIUtility"/>, and restore it back</summary>
+        public class IconSizeScope : IDisposable
+        {
```

Comment:
you should also add documentation or make them internal

Reply:
I saw that `EditorGUIUtility` already has a IconSizeScope :)